### ISSUE TITLE
De-duplicate calls to `convertCrumbType()` within the Chapters component

### ISSUE THREAD

**Issue by ryan953** [getsentry/sentry#34700]
Currently there are two places where we call `convertCrumbType()`

1. Inside of [userActionsNavigator.tsx](https://github.com/getsentry/sentry/blob/9aa7218f605a10c32375263d4eef77279afc2bd5/static/app/views/replays/detail/userActionsNavigator.tsx#L58) we call `transformCrumbs()` which accepts a list of RawCrumb items, and calls `convertCrumbType()` internally.
2. We call `convertCrumbType()` again directly inside [actionCategory.tsx](https://github.com/getsentry/sentry/blob/9aa7218f605a10c32375263d4eef77279afc2bd5/static/app/components/replays/actionCategory.tsx#L37)

Usually Typescript would complain when we pass the wrong argument into a function, but in this case we're passing a `Crumb` where a `RawCrumb` is expected, and `RawCrumb` is an inexact type that Crumb extends. example `type Crumb = RawCrumb & { ... }`

-----

**TL/DR**

This is the spot where `Crumb` is getting converted back into `RawCrumb`:
https://github.com/getsentry/sentry/blob/9aa7218f605a10c32375263d4eef77279afc2bd5/static/app/views/replays/detail/userActionsNavigator.tsx#L128

**Changes Needed:**
1. Update the Prop for `ActionCategory` to be `Crumb`
2. Don't call `convertCrumbType()` from inside `actionCategory.tsx`

----

**Mismatched Types:**

![Image](https://user-images.githubusercontent.com/187460/168870511-0a9510ea-e9ce-47ae-9055-f87d60af870e.png)
![Image](https://user-images.githubusercontent.com/187460/168870512-6b2753ed-6523-491f-ac7e-0efe417cd22c.png)
